public/configs/env.json

```diff
@@ -121,7 +121,7 @@
                }
             ],
             "validators": [
-               "http://35.75.227.118"
+               "http://54.92.118.111"
```

Comment:
Can verify this IP address [here](https://app.terraform.io/app/dydxprotocol/workspaces/validators-dev2/variables).